notebooks/pipeline_management/Neuro Data Pipeline Management.ipynb

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
can you get rid of the spaces in the name of the notebook, those are sometimes problematic in linux

Reply:
Sometimes linux is so useless

Reply:
Will fix it